Cartfile

```diff
@@ -1,2 +1,3 @@
 github "raphaelmor/Polyline" ~> 4.2
 github "mapbox/turf-swift" ~> 0.5
+github "mapbox/mapbox-events-ios" ~> 0.10
```

Comment:
This should be a private dependency, not a public one. Only the example application target depends on the map SDK, which is what depends on MapboxMobileEvents.

Reply:
Thanks, moved `mapbox-events-ios` to `Cartfile.private`.